packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx

```diff
@@ -44,7 +44,7 @@ export class Beak extends BaseComponent<IBeakProps, {}> {
         pointOne = `0, 0`;
         pointTwo = `${BEAK_HEIGHT}, ${BEAK_HEIGHT}`;
         pointThree = `0, ${BEAK_WIDTH}`;
-        transform = 'translateX(100%)';
+        transform = getRTL() ? 'translateX(260%)' : 'translateX(100%)';
```

Comment:
260 is a magic number. It seems a little wrong; shouldn't this be a symmetrical positive/negative 100%? I don't completely undersatnd what's going on here but 260 seems wrong.

Reply:
Found the issue. So we need to RTL-ify the absolute positioning (left/right values) in Coachmark.base.tsx in the _setBeakPosition method